eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -0,0 +1,31 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $serviceName,
```

Comment:
Lets call this `$serviceDirectory` so it is consistent and we understand that we are getting all packages in that directory.